README.md

```diff
@@ -179,7 +179,7 @@ vars:
   ansistrano_ensure_shared_paths_exist: yes
   ansistrano_ensure_basedirs_shared_files_exist: yes
 
-  ansistrano_deploy_via: "rsync" # Method used to deliver the code to the server. Options are copy, rsync, git, svn, s3 or download. Copy, download and s3 have an optional step to unarchive the downloaded file which can be used by adding _unarchive. You can check all the options inside tasks/update-code folder!
+  ansistrano_deploy_via: "rsync" # Method used to deliver the code to the server. Options are copy, copy_zipped, rsync, git, svn, s3 or download. Copy, Copy_zipped, download and s3 have an optional step to unarchive the downloaded file which can be used by adding _unarchive. You can check all the options inside tasks/update-code folder!
```

Comment:
Copy_zipped does not have an optional step... it always happens

Reply:
Do you need unarchiving option flag for it? I think unarchiving always should be because our ansistrano module's goal is deploy project. not backup system.
Give me feedback.

Ah think you for ansistrano module. so I can use ansible to deploy my projects. :-)